assets/src/api.ts

```diff
@@ -39,8 +45,8 @@ const parseRouteData = ({ id, direction_names }: RouteData): Route => ({
 const parseRoutesData = (routesData: RouteData[]): Route[] =>
   routesData.map(parseRouteData)
 
-export const fetchRoutes = (): Promise<Route[]> =>
-  fetch("/api/routes")
+export const fetchRoutes = (userToken?: UserToken): Promise<Route[]> =>
+  request("/api/routes", userToken)
```

Comment:
If I understand correctly, I don't think these requests need to send the userToken anymore?

Reply:
Good call! I removed these and move the token reading logic into the socket code since it is only needed there.